lib/std/winapis.nim

```diff
@@ -0,0 +1,13 @@
+## An replacement for `std/winlean` with precise and stable types definitions.
+
+type
+  DWORD* = culong  ## https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types#ulong
```

Comment:
these should probably not be `distinct`, but what would be pros/cons for making these distinct?

Reply:
I prefer the name `Dword`, we generally depart from the various "coding" styles (where is the style in ugly cargo culting anyway?) and follow the Nim style.